### PR TITLE
Add FAQ's Page

### PR DESCRIPTION
**Pull request by pedrofp4444**

### CODE REVIEW

**Comment by pedrofp4444** [comment]
I liked the result, I'm just having some trouble, because I wanted to, when opening a disclosure, to close the others until I understood that the specific feature I described is predefined but it's not working :disappointed:

**Comment by pedrofp4444** [comment]
Also, how can I surpass the tests once that I used a new component, which means more packages, locally I run ```npm install @headlessui/react```. How can I add them to the tests so that they pass?

**Comment by RuiL1904** [comment]
@pedrofp4444 You need to install the dependency only on the `web` app.
`npm remove @headlessui/react` on the root of the repo and then `npm install @headlessui/react` on the root of the `web` app.

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Here are some question i think you should add:
"Quero ser voluntário! Quais são as opções?" - Ser ou não mentor 
"Como funciona uma sessão?"
"Quais são as responsabilidades e conhecimentos necessários em um voluntário?"

File: apps/web/pages/faqs.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
```
Comment (MarioRodrigues10) [inline apps/web/pages/faqs.js:1]:
```suggestion
```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Could you please convert the ```faqs.js``` page in tsx.
You just need to change the file extension, there will be no conflicts.

**Review by MarioRodrigues10 — APPROVED** [review]

**Pull request merged**